### ISSUE TITLE
Running the ConsoleRunnerTest in the pants repo hangs forever

### ISSUE THREAD

**Issue by ericzundel** [pantsbuild/intellij-pants-plugin#139]
IntelliJ 2016.1.1
Pants Plugin 1.4.1

I used New -> Project from Exising Sources and clicked on the folder tests/java to import a project.

Then I opened up `ConsoleRunnerTest.java` and right clicked on the class `ConsoleRunnerTest` to run it in IntelliJ.  I chose the pants icon for the runner.

Now the UI is stuck saying "Run build" and I'm not getting any other feedback.

<img width="1464" alt="screen shot 2016-05-07 at 11 11 28 am" src="https://cloud.githubusercontent.com/assets/119728/15092881/c5f10300-1444-11e6-8b0e-c2a4c6b3115c.png">

I switched over to the "Messages" tab and it has a message that might be related:

```
 Information:No target found in configuration.
```

<img width="526" alt="screen shot 2016-05-07 at 11 11 41 am" src="https://cloud.githubusercontent.com/assets/119728/15092884/ce3fc6f4-1444-11e6-9431-6726dd14f109.png">

**Comment by wisechengyi** [comment]
likely you just need to click this.
<img width="47" alt="screen shot 2016-05-07 at 10 54 50 am" src="https://cloud.githubusercontent.com/assets/596358/15093629/342d2e46-1442-11e6-8088-d19e10d21755.png">

2016 had it switched to the other mode by default for som reason.

**Comment by ericzundel** [comment]
Yes, I do see output then.  Seems like a pants specific issue.  I've not
had a problem with the native JUnit test runner.

**Comment by wisechengyi** [comment]
It's an IDEA setting issue. Potentially we can set it correctly via the plugin.
